src/app/authorization/registration/registration.component.html

```diff
@@ -1,237 +1,232 @@
-              >
-                Please insert a valid Mobile number.
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="form-group form-check">
-          <div class="form-container">
-            <input
-              type="checkbox"
-              formControlName="acceptTerms"
-              class="form-check-input"
-            />
-            <label for="acceptTerms" class="form-check-label"
-              >Accept Terms & Conditions</label
+      <div class="row">
+        <div class="col-mb-3">
+          <input
+            type="number"
+            placeholder="Mobile Number"
+            class="form-control"
+            id="mobile"
+            formControlName="mobile"
+          />
+          <div class="form-text text-danger fixed-height">
+            <span
+              *ngIf="registrationFormValidator['mobile'].errors?.['required'] && (registrationFormValidator['mobile'].dirty || registrationFormValidator['mobile'].touched)"
+            >
+              Number is required.
+            </span>
+            <span
+              *ngIf="registrationFormValidator['mobile'].errors?.['pattern'] && (registrationFormValidator['mobile'].dirty || registrationFormValidator['mobile'].touched)"
             >
+              Number is Invalid.
+            </span>
+
+            <span
+              *ngIf="registrationFormValidator['mobile'].errors?.['maxlength'] && (registrationFormValidator['mobile'].dirty || registrationFormValidator['mobile'].touched)"
+            >
+              Number is 10 Digits.
+            </span>
+            <span
+              *ngIf="registrationFormValidator['mobile'].errors?.['minlength'] && (registrationFormValidator['mobile'].dirty || registrationFormValidator['mobile'].touched)"
+            >
+              Number is not 10 Digits.
+            </span>
           </div>
         </div>
-        <div class="row">
-          <div class="form-container">
-            <div class="col">
-              <div class="d-grid">
-                <button
-                  type="submit"
-                  [disabled]="!registrationForm.valid"
-                  class="btn btn-success"
-                >
-                  Register Now
-                </button>
-                <p class="text-center mt-2">
-                  Already have an account? <a routerLink="/login">Log In</a>
-                </p>
-              </div>
-            </div>
+      </div>
+      <div class="form-group form-check">
+        <div class="form-container">
+          <input
+            type="checkbox"
+            formControlName="acceptTerms"
+            class="form-check-input"
+          />
+          <label for="acceptTerms" class="form-check-label"
+            >Accept Terms & Conditions</label
+          >
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-mb-3">
+          <div class="d-grid">
+            <button
+              type="button"
```

Comment:
please change this type to submit and do function calling on ngSubmit

Reply:
Ok I have change type of button to submit button